src/malloy/__init__.py

```diff
@@ -25,5 +25,10 @@
 __version__ = "2023.1010"
 
 from malloy.runtime import (Runtime)
+try:
+  from malloy.ipython.ipython_magic import (load_ipython_extension,
+                                            unload_ipython_extension)
+except ModuleNotFoundError:
```

Comment:
Right now I haven't added ipython as a dependency, so this handles that, but maybe we should add it to simplify things?

Reply:
I'm not sure I would add it since it's specific to the one implementation of magics but it not really needed for anything else.

There could also be an argument for extracting the magic out into it's own package independent of the main python malloy package since it's just a consumer of it.

Reply:
I could see it either way. I did a quick scan and its pretty common to include ipython extensions with the main python bundle, but it would make dependencies clearer.